src/main/java/frc/team997/lib/util/DashboardChooser.java

```diff
@@ -48,20 +48,29 @@
     private Logger<T> logger;
     private final boolean logChanges;
 
+    /**
+     * Interface implemented by all enums which DashboardChooser can automatically create a chooser
+     * from.
+     */
     public interface Option {
         String getDisplayName();
     }
 
+    /**
+     * Interface which allows implementing classes to have a method called when a value in
+     * DashboardChooser is changed.
+     */
     public interface ValueUpdater<T> {
         void onOptionSelected(T oldOption, T newOption);
     }
 
     /**
-     * Constructs a DashboardChooser from an enum implementing {@link Option}.
+     * Returns a DashboardChooser from an enum implementing {@link Option}.
```

Comment:
I think Constructs is okay here is basically the java equivalent of a named constructor (the other option is we could make this function a constructor since it has a unique header but I think this is clear